main.js

```diff
@@ -22,22 +23,29 @@ export const httpClient = new Proxy(ky, {
       return propValue;
     }
     return async function() {
```

Comment:
Is this wrapper needed?

Reply:
Yeah, this has to return a function and it can't be a static one, as it needs `this` and `arguments` from `get`.